packages/services/server/src/index.ts

```diff
@@ -443,7 +443,7 @@ export async function main() {
         connectionUri: env.supertokens.connectionURI,
         apiKey: env.supertokens.apiKey,
       },
-      isProduction: env.environment === 'prod',
+      isProduction: env.environment !== 'development',
```

Comment:
We specify `ENVIRONMENT=development` within our .env file for local development, so wont impact us locally.

Reply:
The environment values we use during deployment are:
- `prod`
- `dev`
- `staging`

I wonder whether we actually need them aside from analytics values and instead should make this a `GRAPHQL_INTROSPECTION_ENABLED` flag instead...